qiita_db/util.py

```diff
@@ -969,14 +969,21 @@ def get_processed_params_tables():
         return TRN.execute_fetchflatten()
 
 
-def get_lat_longs():
+def get_lat_longs(portal):
```

Comment:
This was needed to avoid circular import (qiita_db/util.py and qiita_db/portal.py). I could move `get_lat_longs` to meta_util.py